build/isc-bind9/files/named-template.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+
+    This file and its contents are supplied under the terms of the
+    Common Development and Distribution License ("CDDL"), version 1.0.
+    You may only use this file in accordance with the terms of version
+    1.0 of the CDDL.
+
+    A full copy of the text of the CDDL should have accompanied this
+    source. A copy of the CDDL is also available via the Internet at
+    http://www.illumos.org/license/CDDL.
+
+    Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+-->
+
+<service_bundle type='manifest' name='named$(sVERSION)'>
+<service name="network/dns/named$(sVERSION)" type="service" version="1">
+
+<instance name='named$(sVERSION)' enabled='false'>
```

Comment:
The instance name should be `default` here. The version is encoded in the service name.

Reply:
have kept as original, as it is  now similar to nginx and apache, as discussed